test/prototype/test_parq.py

```diff
@@ -273,7 +275,84 @@ def test_intx_weight_only_e2e(self, b: int = 2, group_size: int = 32):
             ProxHardQuant(),
             quant_per_channel=True,
         )
-        self.compare_parq_convert(model, m_ref, optimizer, config)
+        compare_parq_convert(model, m_ref, optimizer, config)
+
+
+class TestStretchedUnifTorchaoQuantizer(common_utils.TestCase):
```

Comment:
New test case that ensures equivalence between PARQ's original `UnifQuantizer` implementation and the new `StretchedUnifTorchaoQuantizer`